src/tunnel.js

```diff
@@ -12,7 +12,7 @@ exports.connect = function({ ngrok, sauce }, tries = 0) {
         username: sauce.username,
         accessKey: sauce.accessKey,
         tunnelIdentifier: sauce.tunnelIdentifier,
-        logfile: path.resolve(__dirname, '../sauce-connect.log'),
+        logfile: path.resolve(__dirname, '../../../sauce-connect.log'),
```

Comment:
This is still relative to where the node_modules are. This should be based on where node is executed from, not where the node_modules are